keyboards/planck/keymaps/3geek14/rules.mk

```diff
@@ -0,0 +1,16 @@
+ifeq ($(strip $(AUDIO_ENABLE)), yes)
+    SRC += muse.c
+endif
+
+CAPS_WORD_ENABLE = yes
+TAP_DANCE_ENABLE = yes
+UNICODEMAP_ENABLE = yes # I maybe need this for send_unicode_string
```

Comment:
Just use `UNICODE_COMMON_ENABLE`

Reply:
I don't see that documented, but I know you were saying on Discord that you're working on Unicode stuff and some of it isn't documented currently. However, when I swap line 7 for
```
UNICODE_COMMON_ENABLE = yes
```
I get this error:
```
Compiling: quantum/keymap_introspection.c                                                          In file included from quantum/keymap_introspection.c:5:
./keyboards/planck/keymaps/3geek14/keymap.c: In function 'process_record_user':
./keyboards/planck/keymaps/3geek14/keymap.c:198:17: error: implicit declaration of function 'send_unicode_string' [-Werror=implicit-function-declaration]
  198 |                 send_unicode_string("┬»\\_(π/┬»"); // Shruggie
      |                 ^~~~~~~~~~~~~~~~~~~
cc1.exe: all warnings being treated as errors
 [ERRORS]
 |
 |
 |
make[1]: *** [builddefs/common_rules.mk:361: .build/obj_planck_ez_glow_3geek14/quantum/keymap_introspection.o] Error 1
Make finished with errors
make: *** [Makefile:392: planck/ez/glow:3geek14] Error 1
```
This was the error I got without Unicode Map. I wasn't sure which version of Unicode to add, but I needed more than 4 hexadigits, so I picked Unicode Map instead of Basic Unicode. (I think UCIS would also work, with a different empty array?)

Reply:
Essentially, the three Unicode input types are:

Basic Unicode = `UC(c)` where `c` is the code point up to 0x7FFF due to keycode space constraints
Unicode Map = `UM(i)` and `UP(i, j)` where `i` and `j` are indexes into the `unicode_map` array, code point can be any valid character
UCIS = replaces typed mnemonics with any Unicode character(s), kinda like the Leader feature. No keycode right now so you need to call `ucis_start()` first

`UNICODE_COMMON_ENABLE` only adds in the underlying Unicode API that these all use (I will probably change this to `UNICODE_ENABLE` and rename that to `UNICODE_BASIC_ENABLE`).

Reply:
Huh, then I'm not sure why the change I tried failed to compile, but it sounds like `UNICODE_ENABLE = yes` should work, allow me to get rid of the empty array, and be slightly more future-proof? I'll try that in the morning.

Reply:
Oh, whoops. It's `UNICODE_COMMON = yes`.

Reply:
That works, thanks!